packages/forma-36-react-components/src/components/Modal/Modal.tsx

```diff
@@ -94,8 +94,8 @@ export function Modal({
   testId,
   topOffset,
   ...otherProps
-}: ModalProps): React.ReactElement {
-  const allProps = {
+}: ModalProps) {
+  const props = {
```

Comment:
the props are being destructured in the argument of the function and then later we putting then together in a `props` variable again

I know this was already there before your PR, but could you fix this here as well?

Reply:
I'd prefer to fix it when we'll be migrating it to v4

Reply:
I'm confused, isn't this PR merging to `forma-v4` branch?

what do you mean by "migrating it to v4" if it's not merging to v4 branch?

Reply:
to correct myself: while migrating the component to Emotion and to a separate package.

Reply:
got it! ok then 😃